src/components/learner-credit-management/search/CatalogSearchResults.jsx

```diff
@@ -88,21 +82,38 @@ export const BaseCatalogSearchResults = ({
   return (
     <div className="mb-5">
       <DataTable
-        isPaginated
-        manualPagination
         columns={courseColumns}
         data={tableData}
+        defaultColumnValues={{ Filter: TextFilter }}
+        initialState={{
+          pageSize: 15,
```

Comment:
[question] I'm setting the initial prop pageSize to display 15 results per the figma mock, but the datatable displays 20. The same behavior happens when I set the prop pageSize on line 98 to 15 as well. Why is this happening?

Reply:
Thanks for the explanation -- I updated based on your feedback.